redisvl/extensions/llmcache/semantic.py

```diff
@@ -103,7 +103,10 @@ def __init__(
         ]
 
         # Create semantic cache schema and index
-        schema = SemanticCacheIndexSchema.from_params(name, prefix, vectorizer.dims)
+        dtype = kwargs.get("dtype", "float32")
```

Comment:
I like that this is `kwargs` here. It's overridable, but not necessary recommended ;)